src/handle-request.js

```diff
@@ -32,31 +35,39 @@ module.exports = (app, request, handlers, done) => {
 
         case 'LaunchRequest':
             callHandler(handlers.onStart, null, request.session.attributes, app, done);
+            info(`Request "${requestType}" handled`);
+            debug(`Request "${requestType}" handled: "${JSON.stringify(request)}"`);
```

Comment:
Move these (from all cases) before the switch
examples:
info(`Received: ${requestType}`)
debug(`Payload: ${JSON.stringify(request, null, 2)}`)

and for intent also (in the case)
info(`Handling intent ${intentName}`)

the default case is alright